server/middlewares/getSingleBlog.js

```diff
@@ -0,0 +1,20 @@
+// import Blog from '../models/blog';
+
+// // Get One middleware
+// async function getBlog(req, res, next) {
+//     let blog;
+//     try{
+//      blog = await Blog.findById(req.params.id);
+//      if(blog == null){
+//          return res.status(404).json({message: 'Sign not found'});
+//      }
+//     } catch(err) {
+//         return res.status(500).json({
+//             error: err.message,
+//         });
+//     }
+//     res.blog = blog;
+//     next();
+// }
+
+// export default getBlog;
```

Comment:
This change (this whole file) shouldn't be here since this the get single request PR.